docs/building-on-etherlink/fees.md

```diff
@@ -1,9 +1,91 @@
 ---
-title: Fee structure
+title: Estimating fees
 ---
 
 import GasPriceWarning from '@site/docs/conrefs/gas-price-warning.md';
 
+The Etherlink gas price (and therefore the fee for a given transaction) varies based on the activity on the chain.
+As activity increases, fees increase, and vice versa.
+
+As described in [Fee structure](#fee-structure), Etherlink fees include the cost of running the transaction and writing the transaction to layer 1 but not a voluntary tip.
+
+## Estimating transaction fees
+
+:::tip
+
+You can use libraries such as [ethers.js](https://docs.ethers.org/v6/) to estimate transaction fees.
+For an example, see [Sending transactions](/building-on-etherlink/transactions).
+
+:::
+
+Etherlink supports the standard EVM `eth_gasPrice` endpoint to provide the current gas price, as in this example:
+
+```bash
+curl --request POST \
+     --url https://node.ghostnet.etherlink.com \
+     --header 'accept: application/json' \
+     --header 'content-type: application/json' \
+     --data '
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "eth_gasPrice"
+}
+'
+```
+
+It returns the gas price plus a safety margin as a hexadecimal number, as in this example:
+
+```json
+{"jsonrpc": "2.0", "result": "0x3b9aca00", "id": 1}
+```
+
+In this response, the hex number `0x3b9aca00` corresponds to the decimal number 1,000,000,000.
+The gas price is given in billionths of one XTZ (as with ETH and gwei), which means that the current gas price is 1 XTZ.
```

Comment:
1 xtz = 1e18 wei
1 gwei = 1e9 wei
1 gwei = 1e-9 xtz

In the example below you have a tx with 623766 gas units & 1 gwei gas price, so the price in xtz is 623766 * 1e-9 = 0.00062377 xtz

Reply:
Thanks for catching that. I don't know if we are using ETH terms like gwei and wei here so I'll try to make it clear without relying on them.

Reply:
I think it's used elsewhere on that page `minimum_base_fee_per_gas: The base fee for Etherlink transactions, which is 1 gwei` and on many pages of the block explorer. XTZ is the native _ether_ token on the chain, so maybe easier to keep it in sync with other EVMs that use similar terms. Check out  https://docs.polygon.technology/tools/gas/polygon-gas-station which uses those terms even though the token is POL